src/vec_avx.h

```diff
@@ -41,7 +41,11 @@
 
 #else
 
+#if defined(_MSC_VER)
```

Comment:
We don't officially support Visual Studio for this version of LPCNet but it doesn't look like this will hurt anything. More of a FYI than anything else.

Reply:
That's a good point.  I am wary about any implied MSVC support.  @jg1uaa can you pls remove Visual Studio support?